data/presets/highway/track.json

```diff
@@ -30,29 +30,34 @@
     "terms": [
         "4wd",
         "4x4",
-        "agricultural road",
         "atv",
-        "carriage road",
         "double track",
-        "farm road",
-        "fire road",
-        "forest road",
         "four wheel drive",
         "high-clearance",
         "jeep",
-        "land access road",
-        "logging road",
         "offroad",
+        "off-road",
         "overgrown",
         "primitive",
         "quad",
-        "ranch road",
         "rut",
-        "track",
-        "track road",
-        "two track",
         "unmaintained",
-        "woods road"
+        "unmaintained road",
+        "dirt road"
```

Comment:
I'd like to keep terms sorted alphabetically. :nerd_face:

Reply:
I didn't realize they were before. So, should I change this?

Reply:
yes, please

Reply:
iD ranks terms by their position in the list, so sorting them alphabetically promotes some rarer terms over more common terms. This might be less of a problem now that some terms are aliases, but it’s hard to test.

Reply:
Are you sure? Is this documented somewhere (+documented for translators)?

Reply:
Oh, either I’m mistaken or it’s no longer in place. It looks like nowadays iD only considers the index of a match [against the preset name](https://github.com/openstreetmap/iD/blob/0c69018f96f4f5e5303c215e03693117f4a4dc1d/modules/presets/collection.js#L93) but only filters by terms.